build.sh.tpl

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+AWS="/usr/local/bin/aws"
+AWS_DEFAULT_REGION=$AWS_DEFAULT_REGION
+RC_DIR=$RC_DIR
+RC_VERSION=$version
+S3_BUCKET=$S3_BUCKET
+
+set -e
+
+# Clean old RC dirs and create new
+function prepare_directories () {
+  sudo rm -rf $RC_DIR-{new,old}
+  sudo mkdir -p $RC_DIR-new
+}
+
+# Download given RocketChat build
+function build_rc {
+    local fname
+    local s3_path
+    fname="rocket.chat-$RC_VERSION.tgz"
+    s3_path="s3://$S3_BUCKET/$fname"
+    cd $(mktemp -d)
+    $AWS s3 cp --quiet $s3_path .
+    tar zxf $fname
+    rm -f $fname
+    npm install --production --prefix ./bundle/programs/server
+    sudo cp ./bundle $RC_DIR-new/
+    sudo chown -R deploy:deploy $RC_DIR-new
```

Comment:
Why do we need chown here? Does `tar` sets non-current user when decompresses by default? Even if so it has the parameter
```
--no-same-owner - Extract files as yourself (default for ordinary users).
```

Reply:
Not sure about that chmod, just put it from fabfile